app/controllers/interviews_controller.rb

```diff
@@ -69,6 +85,29 @@ def set_interview
 
     # Only allow a list of trusted parameters through.
     def interview_params
-      params.require(:interview).permit(:title, :participants_email, :start_time, :end_time)
+      params.require(:interview).permit(:title, :start_time, :end_time, :interviewer_id, :interviewee_id) 
+    end
+
+    def participants_available
+      logger.info((params[:interview]["title"]))
+      start = DateTime.new(params[:interview]["start_time(1i)"].to_i, 
+      params[:interview]["start_time(2i)"].to_i,
+      params[:interview]["start_time(3i)"].to_i,
+      params[:interview]["start_time(4i)"].to_i,
+      params[:interview]["start_time(5i)"].to_i)
```

Comment:
I think ki u dont hv to pass all the parameters like this. check what's coming in `params[:interview]["start_time"]`